apps/spectator/app/_components/GameList/Error.tsx

```diff
@@ -0,0 +1,35 @@
+import { WhistlingIcon } from '@hcc/icons';
+import { Icon } from '@hcc/ui';
+
+import { errorFallback } from './GameList.css';
+
+function GameListErrorFallback({ message }: { message: string }) {
+  return (
+    <div className={errorFallback.root}>
+      <Icon source={WhistlingIcon} size="lg" />
+      <span className={errorFallback.span}>{message}</span>
```

Comment:
의미 있는 이름을 사용하는 것이 어떨까요? 예를 들면 message 같은 이름이 가능하겠네요. 

```suggestion
<span className={errorFallback.message}>{message}</span>
```